### PR TITLE
chore(test): add test cases for getIPsFromString

### PR DESCRIPTION
**Pull request by dshekhar95**
## Problem
test cases missing for getIPsFromString
 

## Solution
add test for empty string, error cases

### CODE REVIEW

File: dgraph/cmd/alpha/run_test.go [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ func TestListTypeSchemaChange(t *testing.T) {
 	res, err = runGraphqlQuery(q)
 	require.NoError(t, err)
 	testutil.CompareJSON(t, testutil.GetFullSchemaHTTPResponse(testutil.
-		SchemaOptions{UserPreds: `{"predicate":"occupations","type":"string"}`}), res)
+	SchemaOptions{UserPreds: `{"predicate":"occupations","type":"string"}`}), res)
```
Comment (dshekhar95) [inline dgraph/cmd/alpha/run_test.go:1079]: accidentally crept in. will revert this

**Review by MichelDiz — APPROVED** [review]

File: dgraph/cmd/alpha/run_test.go [hunk 2]
```diff
@@ -1471,6 +1471,31 @@ func TestIPStringParsing(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEqual(t, net.IPv6zero, addrRange[0].Lower)
 	require.NotEqual(t, addrRange[0].Lower, addrRange[0].Upper)
+
+	addrRange, err = getIPsFromString("")
```
Comment (matthewmcneely) [inline dgraph/cmd/alpha/run_test.go:1475]: Nit: This sort of replication is a good candidate for "table driven tests"; making the test cases easier to read and allowing parallel execution of tests. No need to change this, just an FYI...
